src/commcare_cloud/ansible/roles/postgresql_base/defaults/main.yml

```diff
@@ -20,6 +20,8 @@ postgresql_archive_path: "{{ postgresql_dir_path }}/wal_archive"
 postgresql_archive_command: "cp %p {{ postgresql_dir_path }}/wal_archive/%f"
 postgresql_enable_archive: no
 
+postgresql_systemd_config_dir: "/etc/systemd/system/postgresql@.service.d"
```

Comment:
Is this an OK setting to have on Ubuntu 14.04 as well?

Reply:
it won't have any effect on 14.04